airbyte-integrations/connectors/destination-azure-blob-storage/src/main/java/io/airbyte/integrations/destination/azure_blob_storage/AzureBlobStorageConsumer.java

```diff
@@ -39,6 +46,8 @@
 
   private AirbyteMessage lastStateMessage = null;
 
+  public static final String YYYY_MM_DD_FORMAT_STRING = "yyyy_MM_dd";
```

Comment:
Any reason to have this constant public?

Reply:
I agree with you. Changed access modifier to private.